src/lib/drivers/driver.js

```diff
@@ -86,6 +86,28 @@ class DriverBase {
     return Promise.reject(new Error('Not implemented'));
   }
 
+  evaluateAsync(asyncExpression) {
+    return new Promise((resolve, reject) => {
+      // Inject the call to capture inspection.
+      const expression = `window.__inspect = inspect;${asyncExpression}`;
```

Comment:
what about wrapping `asyncExpression` in an IIFE so `__inspect` doesn't have to live on window and we can avoid the `delete` call?